### PR TITLE
[13.0][MIG] project_parent module

### PR DESCRIPTION
**Pull request by tfossoul**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please preserve commit history following technical method explained in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0.

If the jump is between several versions, you have to modify the source branch in the main command to accommodate it to this circumstance.

**Comment by tfossoul** [comment]
@pedrobaeza  I squash everything into this commit. https://github.com/OCA/project/pull/703/commits/43794a3b5cdcb27ad5c5e0a6602d1eed6f63ed73 because there are 12 commits only for the v12 pull request (https://github.com/OCA/project/pull/644). It is better for readability to get only one.

**Comment by pedrobaeza** [comment]
That's not valid, as readability here is a secondary point, as you haven't respected commit attribution to the original authors of the code, so please preserve such main commit history. You can squash administrative commits with their immediately main commit though.

**Comment by tfossoul** [comment]
The PR in v12 is still not merged. The author didn't keep v10 module history.
So should I keep history from v10 or from the v12 PR ?
When the v12 PR will be merged we will break history.
So the only way to keep history is to ask the v12 PR author to do it, merge its work and then work on this PR v13.

I suppose i'm not the first one to get this kind of problem :/

**Comment by pedrobaeza** [comment]
You can take one of these approaches:

- Take v10 history, and continue from there.
- Take v12 branch, and continue from there.

depending on which code base you are starting from. Obviously, the ideal thing is to get v12 version merge first, but if not possible, you can do this.

**Comment by tfossoul** [comment]
Here is it.

File: project_parent/__manifest__.py
Comment (dreispt) [inline project_parent/__manifest__.py]: I'm not sure there is enough originality to award Authorship.
I would rather have you added to the contributors section, and not here.

**Review by dreispt — APPROVED** [review]

File: project_parent/__manifest__.py
Comment (tfossoul) [reply project_parent/__manifest__.py]: Sure.

**Comment by tfossoul** [comment]
@pedrobaeza @dreispt does we need more approver before been able to merge ?

**Review by HaraldPanten — APPROVED** [review]

**Pull request merged**
